packages/framework/src/Resources/scripts/admin/plugins/jquery.counterup.js

```diff
@@ -51,6 +53,13 @@
                     }
                 }
 
+                // Preserve spaces if input had spaces
+                if (isSpace) {
+                    while (/(\d+)(\d{3})/.test(newNum.toString())) {
+                        newNum = newNum.toString().replace(/(\d+)(\d{3})/, '$1'+' '+'$2');
```

Comment:
can you use `matchAll` and `join(' ')` instead?

Reply:
I am consistent with previous code